teps/0125-configmaps-and-secrets-as-param-value-source.md

```diff
@@ -0,0 +1,189 @@
+
+---
+title: Configmap and Secret as Param Value Source
+authors:
+  - "@rpajay"
+ 
+creation-date: 2022-10-28\
+last-updated: 2022-10-28\
+status: pending
```

Comment:
[Valid status values](https://github.com/tektoncd/community/blob/main/teps/0001-tekton-enhancement-proposal-process.md#tep-metadata) are:

> Must be one of proposed, implementable, implemented,withdrawn, or replaced.

What is meant by "pending"?